documentation/language-reference/domain-modifiers/HTTPS.md

```diff
@@ -0,0 +1,32 @@
+---
+name: HTTPS
```

Comment:
This page needs to be added to `SUMMARY.md`, see [How to add a new page?](https://docs.dnscontrol.org/developer-info/styleguide-doc#how-to-add-a-new-page).